databricks/koalas/indexes.py

```diff
@@ -120,6 +120,37 @@ def size(self) -> int:
         """
         return len(self._kdf)  # type: ignore
 
+    @property
+    def T(self):
```

Comment:
Can you implement `transpose` and reuse it for `T`? `    T = property(transpose)`.

Reply:
ah, it would be better. okay i go for it

Reply:
ah, i just realized that pandas has no `transpose` for `Index`.
then maybe would it be better to keep this behavior??

Reply:
I think pandas has it ..

```python
>>> import pandas as pd
>>> pd.DataFrame({'a': [1,2,3]}).index.transpose()
RangeIndex(start=0, stop=3, step=1)
```

Reply:
oops... then maybe they forgot to upload this to their docs.

<img width="994" alt="스크린샷 2019-11-19 오후 3 50 51" src="https://user-images.githubusercontent.com/44108233/69123367-89445700-0ae4-11ea-9655-88e4c97fa8aa.png">

thanks i'm going to add it !